visualize-admin/visualization-tool
# feat: Merging of cubes UI improvements

Closes #1830

This PR:
- removes termsets from merging of cubes modal dropdown,
- renames "Contains values from" to "Join by" when hovering over a shared dimension tag in merging of cubes modal,
- prefers dual-line and column-line chart before multi-line chart and tries to initialize the dual-line chart with measures from different charts, if other conditions are met (e.g. that dimensions have different units),
- adds an info tooltip to explain what `Concepts` are on the browse page,
- groups filters and dropdown options by cube, in case we have more than one.

## How to test
1. Go to [this link](https://visualization-tool-git-feat-merging-of-cubes-ui-imp-b426b5-ixt1.vercel.app/en/browse?dataSource=Prod).
2. ✅ Scroll a bit down to see that there's an icon tooltip next to `Concepts` navigation menu.
3. Search for "photo" and select `Einmalvergütung für Photovoltaikanlagen` dataset.
4. Start a visualization.
5. ✅ Go to Vertical Axis menu and open a dropdown. See that the popup menu looks the same as before.
6. Go back to previous view (`Back to main`).
7. Click on `Add dataset`.
8. ✅ Open dimensions dropdown and see that termsets are not longer visible in the popup menu.
9. See a second row (`Median electricity tariff per canton` dataset) and hover over "Canton" chip.
10. ✅ See that the text says "Joined by" Cantons instead of "Contains values from" Cantons.
11. Scroll down a bit and proceed with merging the `International comparison: Expenditure of general government by function (COFOG)` dataset.
12. ✅ See that after merging, chart type changed to dual-axis line chart and that selected dimensions come from different cubes.
13. ✅ Scroll down the left panel and see that the filters are grouped by cubes.
14. Switch chart type to column chart.
15. Open Horizontal Axis field.
16. ✅ Open dropdown and see that dimensions are grouped by cube (including indication of `joinBy` dimensions).
